Phonebook.Backend/Phonebook.Backend.PictureService/Phonebook.Backend.PictureService/Startup.cs

```diff
@@ -0,0 +1,174 @@
+            services.AddSingleton<PictureServiceConfiguration>(AppSettings);
+            services.UseConfigurationValidation();
+            services.ConfigureValidatableSetting<PictureServiceConfiguration>(Configuration);
+            services.ConfigureValidatableSetting<ContactInformation>(Configuration.GetSection("ContactInformation"));
+            services.AddSingleton<PictureServiceConfiguration>(AppSettings);
+
+            // Add Api Versioning Logic
+            services.AddApiVersioning(o =>
+            {
+                o.ReportApiVersions = true;
+                o.AssumeDefaultVersionWhenUnspecified = true;
+                o.DefaultApiVersion = new ApiVersion(1, 0);
+
+            });
+
+            services.AddVersionedApiExplorer(options =>
+            {
+                // add the versioned api explorer, which also adds IApiVersionDescriptionProvider service
+                // note: the specified format code will format the version as "'v'major[.minor][-status]"
+                options.GroupNameFormat = "'v'VVV";
+                // note: this option is only necessary when versioning by url segment. the SubstitutionFormat
+                // can also be used to control the format of the API version in route templates
+                options.SubstituteApiVersionInUrl = true;
+            });
+
+            //Enable IIS Integration            
+            services.AddAuthentication(IISDefaults.AuthenticationScheme);
+            services.Configure<IISOptions>(options =>
+            {
+                options.AutomaticAuthentication = true;
+            });
+            services.AddCors(o => o.AddPolicy(CorsPolicy, builder =>
+            {
+                // Add some CORS allowed origins here
+                builder.WithOrigins(this.AppSettings.AllowedCORSDomains).AllowAnyMethod().AllowAnyHeader().AllowCredentials();
+            }));
+
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+
+
+
+            services.AddImageProcessingSettings(Configuration);
+
+            // Scheduled Tasks
+            services.AddSingleton<IScheduledTaskOptions<PurgeTask>>(
+                new ScheduledTaskOptions<PurgeTask>
+                {
+                    // Once every day at startup -> May be exectuted multiple times a day depending on recycling
+                    Schedule = AppSettings.PurgeSchedule
+                }
+            );
+            services.AddScheduledTask<PurgeTask>();
+            services.AddHostedService<SchedulerHostedService>();
+
+
+            // Add Swagger 
+            services.AddTransient<IConfigureOptions<SwaggerGenOptions>, ConfigureSwaggerOptions>();
+            services.AddSwaggerGen(c =>
+            {
+                // add a custom operation filter which sets default values
+                c.OperationFilter<SwaggerDefaultValues>();
+                
+                //c.SwaggerDoc("v1", new Info

```

Comment:
If this is not needed anymore please remove it

Reply:
Done.